runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -349,6 +352,33 @@ public void write(IToken token, Object data) throws OverwriteException {
                     .peek(e, address));
     }
 
+    /**
+     * Commit the addresses in the range by first inspecting the addresses
+     * and if data not exist in log, hole fill the address.
```

Comment:
nit: ... if data does not exist in the log, ...

Reply:
Done.